docker-hub/download-rate-limit.md

```diff
@@ -26,8 +26,46 @@ manifest requests.
 - Limits are applied based on the user doing the pull, and 
 not based on the image being pulled or its owner.
 
-Docker will gradually introduce these rate limits, with full
-effects starting from November 1st, 2020.
+Docker will gradually introduce these rate limits starting November 2nd, 2020.
+
+## How do I know my pull requests are being limited
+
+When you issue a pull request and you are over the limit for your account type, Hub will return a `429` response code with the following body when the manifest is requested:
+> You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limits
+
+You will see this error message in the Docker CLI or Docker Engine logs.
+
+## How can I check my current rate
+
+Valid, non-rate-limited manifest API reqests to Hub will include the following rate limit headers in the response:
+
+> RateLimit-Limit    
+RateLimit-Remaining
+
+If you have a proxy or other layer in place that logs your requests, you can inspect the headers of these responses directly.
+
+Otherwise, you can use curl to view these. You will need `curl`, `grep`, and `jq` installed.
+
+To get a token anonymously (if you are pulling anonymously):
+
+> $ TOKEN=$(curl "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
+
+To get a token with a user account (if you are authenticating your pulls) - insert your username and password in the following command:
+
+> $ TOKEN=$(curl --user 'username:password' "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
+
+Then to get the headers showing your limits, run this:
+
+>$ curl -v -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
+
+Which should return something like this:
+
+> < RateLimit-Limit: 100;w=21600  
+< RateLimit-Remaining: 76;w=21600
+
+This means my limit is 100 per 21600 seconds (6 hours), and I have 76 pulls remaining.
+
+Please remember that these headers are best-effort and there will be small variations.
```

Comment:
We should have a disclaimer that this ends up using up your pulls every 2nd request. (Is that intentional?)

Reply:
It is intentional because it is actually counting manifest requests and multi-architecture images can result in two manifests requests to actually pull the image, so we double the counter to make up for it.